components/Text.jsx

```diff
@@ -1,7 +1,13 @@
 import Interweave from 'interweave';
 
-const Text = ({ block }) => (<section className="grid-container usa-section" id="share">
+const Text = ({ block }) => (<section className="grid-container usa-section">
   <div className="grid-row grid-gap">
+    {
+      block.image && block.data.includes('image:left') &&
```

Comment:
I wonder if we made a separate "text with image" block that would serve our users better. I'm nervous about them needing to do a lot with the data field (especially since it's at the end of the CMS) and just changing the type seems easier

Reply:
I actually had TextWithImage.jsx created and decided against it, because that was already going to need a data param to specify whether the image should go on the left and right... and was thus a superset of the Text functionality. So I think it's simpler to be able to add an image to your block-text block, and add a data to explain which side

OH maybe most foolproof is to have images default to left side (if no data) and you can put them on the right if you know the correct data incantation "image:right"